### ISSUE TITLE
AttributeError: 'CompilationUnit' object has no attribute 'bytecode_runtime

### ISSUE THREAD

**Issue by wwwqql** [gcf3711/sGuardPlus#3]
按照readme流程进行运行，在执行智能合约修复命令时node index.js ../example/motivation_example.sol
出现AttributeError: 'CompilationUnit' object has no attribute 'bytecode_runtime，请问我该怎么解决。
![image](https://github.com/user-attachments/assets/6c39d80e-7941-433e-aee6-88f626a7bfdf)

**Comment by thomasyonug** [comment]
The errors are raised by Slither. 
Since the toolchain version of your environment is unknown, it's plausible that the version of solc is not supported by Slither. 
Please check it or repost the environment information here.

**Comment by wwwqql** [comment]
非常感谢的回复，我的环境中slither-analyzer为0.8.1，Python 版本3.8.10，Solidity 编译器版本为0.4.26.其余包均按照requirements.txt文档要求安装，只有excrytic-compile==0.2.2没有安装成功。请您帮我检查那个地方导致报错。您的建议将帮助我解决问题。
![image](https://github.com/user-attachments/assets/3bf596dd-7fe8-46b6-a772-b0825fd65cee)

**Comment by Tanggerr** [comment]
you should replace ”excrytic-compile==0.2.2“ to "crytic-compile==0.2.2".

**Comment by wwwqql** [comment]
非常感谢您的建议，安装crytic-compile==0.2.2后，可以运行了。这是已经修复合约了吗，修复后的合约保存在motivation_example.sol.fixed 还是motivation_example.sol。
![image](https://github.com/user-attachments/assets/ac3b1472-7fbc-4028-9b84-78374d1d0c95)

**Comment by Tanggerr** [comment]
yeah,it has fixed successfully,and the motivation_example.sol.fixed is the fixed solidity.
